Assets/Content/Furniture/NetworkedOpenable.cs

```diff
@@ -22,9 +26,36 @@ public override IInteraction[] GenerateInteractionsFromTarget(InteractionEvent i
             OpenInteraction openInteraction = new OpenInteraction();
             openInteraction.icon = OpenIcon;
             openInteraction.OpenStateChange += OnOpenStateChange;
+            openInteraction.OpenStateChangeSendEvent += OnOpenStateChangeReceiveEvent;
             return new IInteraction[] { openInteraction };
         }
-        
+
+        /// <summary>
+        /// Method called whenever a NetworkedOpenable object is opened or closed. 
+        /// </summary>
+        public void OnOpenStateChangeReceiveEvent(object sender, OpenInteractionEventArgs e)
+        {
+            CloseUIWhenClosed(e);
+        }
+
+        /// <summary>
+        /// Close the UI of opened containers when they are closed
+        /// </summary>
+        private void CloseUIWhenClosed(OpenInteractionEventArgs e)
+        {
+            if (e.interactionEvent.Source is Hands)
+            {
+                Hands hands = e.interactionEvent.Source as Hands;
+
+                if (e.interactionEvent.Target is OpenableContainer && !e.Open)
+                {
+                    OpenableContainer openableContainer = e.interactionEvent.Target as OpenableContainer;
+                    AttachedContainer attachedContainer = openableContainer.GameObject.GetComponent<AttachedContainer>();
```

Comment:
You should be able to get the container from the openable (`GetComponent` is not okay here as there can be multiple).

Reply:
Currently multiple containers are not possible on a single object the way the container system is laid out, partly because there's similar lines of code in other files. My suggestion would be to just add a warning comment for now, and wait for the container redesign to handle properly multiple containers at once.